src/app/styles/navbar/navbar.module.css

```diff
@@ -12,13 +12,17 @@
     align-items: center;
     height: 100%;
     position: relative;
+    font-family: "Lato";
```

Comment:
less performant :( causes duplicate download of font.

Reply:
Will fix!